internal/pb/export/export.proto

```diff
@@ -67,24 +76,44 @@ message TemporaryExposureKey {
   optional bytes key_data = 1;
 
   // Varying risks associated with exposure depending on type of verification
-  optional int32 transmission_risk_level = 2;
+  // Ignored by the v1.5 API when report_type is set.
+  optional int32 transmission_risk_level = 2 [deprecated = true];
 
   // The interval number since epoch for which a key starts
   optional int32 rolling_start_interval_number = 3;
 
   // Increments of 10 minutes describing how long a key is valid
-  optional int32 rolling_period = 4
-      [default = 144];  // defaults to 24 hours
+  optional int32 rolling_period = 4 [default = 144];  // defaults to 24 hours
+
+  // Data type representing why this key was published.
+  enum ReportType {
+    UNKNOWN = 0;  // Never returned by the API.
```

Comment:
optional nit: we might leave out the "// Never returned by the API" comments in this repo, which is server focused.

Reply:
I'd rather leave this in as each of these comments got meticulously covered with Apple.
(And less diff to manage between copies)

Reply:
Are we guaranteeing that the server will never publish a key w/ UNKNOWN?

Reply:
There's an affordance to let the client specify how unsupported reportTypes are handled, so I don't think we need to make any particular guarantee for the server.

Reply:
ok, the opposite of Guray's nit - we should specify that the server will publish keys w/ status UNKNOWN if we receive them.

Reply:
There's a conflation here about which api we're talking about. The api this comment refers to is the EN API on device, right? That's why I was proposing removing the comment here. Or we could specify which api.

Reply:
Clarified.